plugins/grpc/go/v1.5.0/buf.plugin.yaml

```diff
@@ -0,0 +1,22 @@
+version: v1
+name: buf.build/grpc/go
+plugin_version: v1.5.0
+source_url: https://github.com/grpc/grpc-go
+integration_guide_url: https://grpc.io/docs/languages/go/quickstart
+description: Generates Go client and server stubs for the gRPC framework.
+output_languages:
+  - go
+deps:
+  - plugin: buf.build/protocolbuffers/go:v1.34.2
+registry:
+  go:
+    min_version: "1.19"
```

Comment:
I wonder if we need to bump this because of these changes:

https://github.com/grpc/grpc-go/pull/7282#issuecomment-2142796379

Reply:
I think you're right, it also says `1.21` [here](https://github.com/grpc/grpc-go/blob/master/cmd/protoc-gen-go-grpc/go.mod)

Reply:
That'll be for the binary, we typically don't care about that module, but the runtime is on `go1.21`

https://github.com/grpc/grpc-go/blob/master/go.mod#L3

Updated in https://github.com/grpc/grpc-go/pull/7282

Reply:
Oh I think I understand it now. So whoever consumes it as remote SDK would need 1.21 or higher.

Reply:
On second thought, we can leave this because we're pulling in `google.golang.org/grpc@v1.62.2` and the minimum for that is `go1.19`